cypress/support/generate.js

```diff
@@ -0,0 +1,12 @@
+const faker = require('faker');
+
+function generateUser() {
+  const random = Math.random().toString().slice(2, 6);
+  const username = faker.internet.userName() + '_' + random;
```

Comment:
You are using '+' operator to concatenate username and random number. It would be better to use template literals for string concatenation. This makes the code cleaner and easier to read.

Reply:
I did not write this code. I only use it. But ok, I can do some refactoring.